src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -0,0 +1,298 @@
+export function selectActiveExpressCareFacility(state, nowMoment) {
+  const current = selectActiveExpressCareWindows(state, nowMoment);
+
+  if (!current?.length) {
+    return null;
+  }
+
+  return {
+    facilityId: current[0].facilityId,
+    siteId: current[0].facilityId.substring(0, 3),
+  };
+}
+
+function getFormattedTime(time) {
+  return moment(`${moment().format('YYYY-MM-DD')}T${time}:00`).format('h:mm a');
+}
+
+function getWindowString(window, timezoneAbbreviation, isToday) {
+  return `${isToday ? 'today' : titleCase(window.day)} from ${getFormattedTime(
+    window.startTime,
+  )} to ${getFormattedTime(window.endTime)} ${timezoneAbbreviation}`;
+}
+
+/**
+ * Returns next schedulable window.  If today is schedulable and current time is before window,
+ * return today's window.  Otherwise, return the next schedulable day's window
+ */
+export function selectNextAvailableExpressCareWindowString(state, nowMoment) {
+  const supportedFacilities = selectExpressCareFacilities(state);
+  if (!supportedFacilities?.length) {
+    return null;
+  }
+
+  const facility = supportedFacilities[0];
+  const siteId = facility.facilityId.substring(0, 3);
+  const { timezone } = getTimezoneBySystemId(siteId);
+  const timezoneAbbreviation = getTimezoneAbbrBySystemId(siteId);
+  const nowFacilityTime = nowMoment.clone().tz(timezone);
+  const dayOfWeek = nowFacilityTime.format('dddd').toUpperCase();
+  const todayDayOfWeekIndex = Number(nowFacilityTime.format('d'));
+  const todaysWindow = facility.days.find(d => d.day === dayOfWeek);
+
+  // Sort schedulable days after today so we can easily find the next
+  // day if needed
+  const schedulableDaysAfterToday = [
+    ...facility.days.filter(day => day.dayOfWeekIndex > todayDayOfWeekIndex),
+    ...facility.days.filter(day => day.dayOfWeekIndex < todayDayOfWeekIndex),
+  ];
+
+  if (todaysWindow) {
+    const start = moment.tz(
+      `${nowFacilityTime.format('YYYY-MM-DD')}T${todaysWindow.startTime}:00`,
+      timezone,
+    );
+    if (nowMoment.isBefore(start)) {
+      // If today is schedulable and we are before the window, return today's window
+      return getWindowString(todaysWindow, timezoneAbbreviation, true);
+    } else {
+      // In the rare case the today is the only schedulable day and they are past the window,
+      // return today's window and specify "next week". Otherwise, return the next schedulable day
+      if (!schedulableDaysAfterToday.length) {
+        return `next ${getWindowString(
+          todaysWindow,
+          timezoneAbbreviation,
+          false,
+        )}`;
+      }
+      return getWindowString(
+        schedulableDaysAfterToday[0],
+        timezoneAbbreviation,
+        false,
+      );
+    }
+  } else {
+    // If today isn't schedulable, return the next day that is
+    return getWindowString(
+      schedulableDaysAfterToday[0],
+      timezoneAbbreviation,
+      false,
+    );
+  }
+}
+
+export function selectExpressCareAvailability(state) {
```

Comment:
The current `selectExpressCare` selector mixed together availability info we keep in `state.appointments` with state from `state.expressCare` and was used mostly in the appointment list code.

So, I've renamed this selector and removed the use of `state.expressCare` in here. For express care pages that need the state from `state.expressCare`, there's a `selectExpressCare` selector in the express care selectors.